framework/test/mocha/unit/modules/chain/blocks/block.js

```diff
@@ -294,6 +351,37 @@ describe('block', () => {
 	});
 
 	describe('create', () => {
+		describe('createV1', () => {
```

Comment:
Maybe it makes sense to have two different test files for checking block v1 and v2?

Reply:
I agree, the the unit test must follow the exact same file structure as source files.

Reply:
Addressed this comment, thanks! :)